### DISCUSSION TITLE
Trinidad and Tobago Television

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#733] [Q&A]
**Channel Name:** Trinidad and Tobago Television (TTT)
**Channel ID:** TTT.tt
**Country:** Trinidad and Tobago
**Language:** English
**Website:** https://webplayer.tegotv.com/tego-live-tv.php?channel=255
**Stream URL (optional):** N/A
**Notes (optional):** Wikipedia article: https://en.wikipedia.org/wiki/Trinidad_and_Tobago_Television Note: Like Trinidad and Tobago channels, this is broadcasting from TegoTV service with token.

**Comment by BellezaEmporium** [comment]
https://webplayer.tegotv.com/wms_auth.php POST with good UID then paste to link afterwards.

Might make an all in 1 solution if possible (if streams are defined by uid on streaming server, perfect for me)

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by BellezaEmporium** [comment]
> https://webplayer.tegotv.com/wms_auth.php POST with good UID then paste to link afterwards.
> 
> Might make an all in 1 solution if possible (if streams are defined by uid on streaming server, perfect for me)

Will try to make a plugin for this one.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/tt.m3u#L12-L13
